Uspihh.Services/Services/IService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace Uspihh.Services.Services
+{
+    public interface IService<T>
+    {
```

Comment:
Great idea to use Generic interface.
In this case we should add method Create(T entity) and, may be, Delete(T entity) to this interface and imlement it in each service. The same with method GetAll